Tasks/google-play-release/GooglePlay.js

```diff
@@ -308,12 +303,33 @@ function addAllChangelogs(languageCode, directory) {
             return pathIsFile;
         });
 
-        for (var i in changelogs) {
-            var fullChangelogPath = path.join(changelogDir, changelogs[i]);
+        var versionCodeFound = false;
+        changelogs.forEach(function(changelogFile) {
+            var changelogName = path.basename(changelogFile, path.extname(changelogFile));
+            var changelogVersion = parseInt(changelogName, 10);
+            if (apkVersionCodes.indexOf(changelogVersion) == -1) {
+                tl.debug(`File ${changelogFile} is not a valid version code`);
+                return;
+            }
+
+            versionCodeFound = true;
+            var fullChangelogPath = path.join(changelogDir, changelogFile);
             addAllChangelogsPromise = addAllChangelogsPromise.then(function (changelog) {
                 console.log(`Appending changelog ${changelog}`);
-                return addChangelog.bind(this, languageCode, changelog)();
+                return addChangelog.bind(this, languageCode, changelog, changelogVersion)();
             }.bind(this, fullChangelogPath));
+        });
+
+        // if there's a single change log file, and it doesn't match a version code, assume it's a global change log
+        if (!versionCodeFound && changelogs.length == 1) {
```

Comment:
This use case was already supported before my changes, but it didn't work for multiple packages. In addChangelog, if the change log file name could not be parsed as int, the global APK version code was used instead. In case of multiple APK, the global APK version code doesn't work.